### PR TITLE
fix(notification): fix explanation mark color

### LINKED ISSUE #3216: [Notification] Dark theme icon color wrong
In warning notifications, center exclamation mark should be **black in all themes**. We do not have a token with such value. This is okay to be hard coded in this situation. 

image

### PR DESCRIPTION
**Pull request by asudoh**
Fixes #3216.

#### Changelog

**Changed**

- Fixed explanation mark icon color in notification.

#### Testing / Reviewing

Testing should make sure our notification is not broken.

@carbon-design-system/design: I chose gray 100 to match text color in dark theme, but please let me know if we should choose literally black or pick another color. Thanks!

### CODE REVIEW

**Review by emyarod — APPROVED** [review]
looks good to me, pending design approval

**Review by laurenmrice — APPROVED** [review]
looks good to me !

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Wait actually, the value needs to be Black. not Gray 100

**Comment by joshblack** [comment]
@laurenmrice is there a token for this or nah?

**Comment by laurenmrice** [comment]
@joshblack I don't believe there is a token for a black value. We are using pure black fills for these instead of Gray 100 to make this as accessible as possible (according to Sadek).

**Comment by joshblack** [comment]
Oh sorry @laurenmrice! Didn't see the black note. Makes sense! I believe we have `black-100` or `carbon--black-100` for this 👍

**Comment by laurenmrice** [comment]
ok great! thanks @joshblack

**Comment by asudoh** [comment]
Thanks @laurenmrice for reviewing! Fixed.

**Review by laurenmrice — COMMENTED** [review]
Warning notification icon fill looks good now for both toast and inline notifications 👍. However I am noticing we need to have a black fill for the check and error icon as well for these light themes. Would you want to tackle that in this pr or make a separate one?

Screen Shot 2019-07-17 at 11 33 50 AM

**Review by joshblack — APPROVED** [review]

File: packages/components/src/components/notification/_inline-notification.scss
Comment (joshblack) [inline packages/components/src/components/notification/_inline-notification.scss]:
```suggestion
    fill: $carbon__black-100;
```

File: packages/components/src/components/notification/_toast-notification.scss
Comment (joshblack) [inline packages/components/src/components/notification/_toast-notification.scss]:
```suggestion
    fill: $carbon__black-100;
```

**Review by joshblack — DISMISSED** [review]
`ibm-color` variables are deprecated

**Comment by jeanservaas** [comment]
@asudoh @laurenmrice just talked to @joshblack and he says it makes sense to keep the requested  changes to the checkmark and the error icons in this PR too since they all pertain to notifications.

So can we make these updates before we merge this PR too:

- [ ] need black fill for error icon for accessibility

- [ ] need black fill for check mark icon for accessibility

image

**Comment by joshblack** [comment]
Yeah, seems like we could tackle those along with these changes if that's okay @asudoh?

**Comment by asudoh** [comment]
Let's track extra change in a separate issue. Adding requirement in PR review process tends to make PRs lingering.

**Review by jeanservaas — APPROVED** [review]
LGTM will move these other icon changes to a seperate issue

**Review by laurenmrice — APPROVED** [review]

**Comment by jeanservaas** [comment]
new issue is tracked here, we can merge this PR

https://github.com/carbon-design-system/carbon/issues/3452

**Pull request merged**
